### PR TITLE
Doc added for the - AutoComplete not Disabled

### PR DESCRIPTION
**Pull request by vinay033**
### :chains: Related Resources
https://chefio.atlassian.net/browse/CHEF-7272
https://chefio.atlassian.net/browse/CHEF-12393
### :+1: Definition of Done
Added DOC changes.
### :white_check_mark: Checklist

**All PRs** must tick these:

- [ ] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [ ] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [ ] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [ ] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [ ] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [ ] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [ ] Tests added/updated? (all new code needs new tests)
- [ ] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### CODE REVIEW

File: components/docs-chef-io/content/automate/security_best_practices.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+gh_repo = "automate"
+[menu]
+  [menu.automate]
+    title = "Security Best Practices"
+    parent = "automate/reference"
+    identifier = "automate/reference/security_best_practices.md Security Best Practices"
+    weight = 50
++++
+
+## Insuring autocomplete remains disable on login screen.
```
Comment (dishanktiwari2501) [inline components/docs-chef-io/content/automate/security_best_practices.md:15]: Replace it with: ## Insuring autocomplete remains disabled on the login screen.

File: components/docs-chef-io/content/automate/security_best_practices.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+Their is a configration maintained in automate this determine autocomplete functionlity
```
Comment (dishanktiwari2501) [inline components/docs-chef-io/content/automate/security_best_practices.md:17]: Replace it with: A configuration maintained in automate determines to autocomplete functionality in the nginx config.

File: components/docs-chef-io/content/automate/security_best_practices.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+in the nginx config
```
Comment (dishanktiwari2501) [inline components/docs-chef-io/content/automate/security_best_practices.md:19]: remove this

File: components/docs-chef-io/content/automate/security_best_practices.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+Their is a configration maintained in automate this determine autocomplete functionlity
+
+in the nginx config 
+
+    
+    add_header Cache-Control "no-cache, no-store, must-revalidate";
+    
+
+Please insure that you don't update or change this confugration.
```
Comment (dishanktiwari2501) [inline components/docs-chef-io/content/automate/security_best_practices.md:25]: Replace it with: Please ensure you don't update or change this configuration.

**Review by dishanktiwari2501 — APPROVED** [review]
Please make the changes suggested.

File: components/docs-chef-io/content/automate/security_best_practices.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+gh_repo = "automate"
+[menu]
+  [menu.automate]
+    title = "Security Best Practices"
+    parent = "automate/reference"
+    identifier = "automate/reference/security_best_practices.md Security Best Practices"
+    weight = 50
++++
+
+## Insuring autocomplete remains disabled on the login screen.
```
Comment (vkarve-chef) [inline components/docs-chef-io/content/automate/security_best_practices.md:15]: @vinay033 could you please fix a typo here? `Ensuring autocomplete...`

File: components/docs-chef-io/content/automate/security_best_practices.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+A configuration maintained in automate determines to autocomplete functionality in the nginx config.
```
Comment (vkarve-chef) [inline components/docs-chef-io/content/automate/security_best_practices.md:17]: @vinay033 , @dishanktiwari2501  can we reword as- A configuration maintained in Chef Automate's `nginx.conf` file determines the autocomplete functionality.

**Pull request merged**